pennylane/plugins/default_qubit.py

```diff
@@ -114,8 +115,12 @@ def apply(self, operations, rotations=None, **kwargs):
                 basis_state = par[0]
                 self.apply_basis_state(basis_state, wires)
 
-            elif isinstance(operation, MultiRZ):
-                self._state = self.vec_vec_product(operation.eigvals, self._state, wires)
+            elif isinstance(operation, DiagonalOperation):
+                self._state = self.vec_vec_product(operation.diagonal, self._state, wires)
+
+            elif len(wires) <= 2:
+                # Einsum is faster for small gates
+                self._state = self.mat_vec_product_einsum(operation.matrix, self._state, wires)
```

Comment:
Just checking if the changes in this file are accidental commits? They appear out of scope for this PR.

Reply:
Yes, they are accidental. I was switching back and forth between this and the default qubit performance branch and once I messed up and had to reset to a commit. This mixed up stuff sadly...